src/cloud/docker/docker-quick-reference.md

```diff
@@ -81,8 +81,10 @@ Options:
       --no-mailhog                                     Disable MailHog
       --mailhog-http-port                              Custom HTTP port for MailHog
       --mailhog-smtp-port                              Custom SMTP port for MailHog
-      --set-docker-host                                Sets host.docker.internal for fpm_xdebug container to
-                                                       resolve debug issue for LINUX system
+      --set-docker-host_xdebug                         Sets host.docker.internal for fpm_xdebug container
```

Comment:
I discourage providing long CLI help lists in the guide because it causes maintenance. It's better to have the reader run the CLI command for their available options.

Reply:
I agree. Way too hard to maintain as this was just one of the diffs I spotted. I'll delete everything and emphasize running the command to see the latest help.